pkg/cloud/api/check_test.go

```diff
@@ -272,3 +274,155 @@ func TestCheckSchema(t *testing.T) {
 		})
 	}
 }
+
+func TestConvertToT(t *testing.T) {
```

Comment:
we need 

* slice -> pointer
* slice -> slice 
* map[] -> pointer

Reply:
done